scripts/core/editor3/helpers/highlights.js

```diff
@@ -88,6 +88,10 @@ function setHighlightsState(editorState, hightlightsState) {
     return setCustomDataForEditor(editorState, editor3DataKeys.MULTIPLE_HIGHLIGHTS, hightlightsState);
 }
 
+export function isHighlightType(styleName) {
+    return styleName.lastIndexOf('-') !== -1;
```

Comment:
Can we use `highlightTypeValid` from `core/scripts/core/editor3/helpers/highlights.js` ?